Source/Controllers/SPAppController.m

```diff
@@ -213,7 +213,7 @@ - (void)applicationDidFinishLaunching:(NSNotification *)notification
 
         SPLog(@"We have stale bookmarks");
 
-        NSString *staleBookmarksString = [secureBookmarkManager.staleBookmarks componentsJoinedByString:@"\n"];
+        NSString *staleBookmarksString = [secureBookmarkManager.staleBookmarks componentsJoinedByString:@"\n\n"];
 
         [NSAlert createDefaultAlertWithTitle:[NSString stringWithFormat:NSLocalizedString(@"App Sandbox Issue", @"App Sandbox Issue")]
                                      message:[NSString stringWithFormat:NSLocalizedString(@"You have stale secure bookmarks:\n\n%@\n\nWould you like to re-request access now?", @"Would you like to re-request access now?"), staleBookmarksString]
```

Comment:
Perhaps we may want to a little info here to explain what bookmarks are, or a link to a docs page on the website explaining them?

Reply:
Writing some docs now.

Reply:
https://sequel-ace.com/ref/bookmarks.html

Reply:
https://user-images.githubusercontent.com/1576190/103556033-1d8d9b00-4eec-11eb-9060-29f4e118c64f.mp4

Reply:
<img width="420" alt="Screenshot 2021-01-05 at 12 11 28 AM" src="https://user-images.githubusercontent.com/1576190/103556184-5cbbec00-4eec-11eb-83b4-bb7f35b26a0e.png">